.github/workflows/release.yml

```diff
@@ -51,7 +51,6 @@ jobs:
           ./gradlew -p codegen/sdk-codegen stageSdks
           ./gradlew --stop
           ./scripts/mergeModels.sh Sources/Services
-      - name: Generate Package.swift
```

Comment:
this should be
```
- name: Generate Package.swift
   run: |

Reply:
I simply removed it for now, as it was the minimum change that allows the workflow to parse correctly, and it reverts this workflow to the state it was in for the previous release which was known to work correctly.

Reply:
but forces to share the env var across steps, we don't want to pass env var declared in Build and Test with Generate Package.swift

Reply:
Understood.  Splitting this into a separate step so `CI_DIR` env vars aren't set when the release `Package.swift` is generated

Reply:
nit: we are only Generating the manifest, SDK generation has been done in earlier step.